app/views/fragments/checkout/fieldsPayment.scala.html

```diff
@@ -1,11 +1,15 @@
 @(form: Form[model.SubscriptionData])
 
-<div class="form-field js-checkout-account">
+@hasError(key: String) = @{
```

Comment:
If you find yourself repeating this kind of code too much, remember that you can always create helper methods under the `views` package, and they will be available in the view without the need of explicitly importing them